scripts/base/protocols/rdp/main.zeek

```diff
@@ -188,6 +188,13 @@ event rdp_client_core_data(c: connection, data: RDP::ClientCoreData) &priority=5
 	set_session(c);
 
 	c$rdp$keyboard_layout       = RDP::languages[data$keyboard_layout];
```

Comment:
This could actually use the `in` operator to reduce some of the calls to `fmt`.

Something like:

```
if ( data$keyboard_layout in RDP::languages )
	{
	c$rdp$keyboard_layout = RDP::languages[data$keyboard_layout];
	}
else
	{
	c$rdp$keyboard_layout = fmt("%s (Best Guess)", RDP::languages[data$keyboard_layout] & 0xffff]);
	}
````

See if that does what you're expecting from it.